app/src/main/res/values/strings_event.xml

```diff
@@ -4,6 +4,8 @@
     <string name="event_location">Location</string>
     <string name="event_distance">Distance</string>
     <string name="event_date_and_time">Date and time</string>
+    <string name="event_date">Date</string>
```

Comment:
I think we can remove this if it is replaced by `<string name="event_date_and_time">Date and time</string>`
```
<string name="event_date">Date</string>
<string name="event_time">Time</string>
```